MetaMask/metamask-extension
# Replace hardcoded sent ether label on confirm screen

Fixes #10974
Fixes issue where `sent ether` label appears on confirm screen for sends of non-ether native currencies of custom networks:
<img width="1020" alt="Screen Shot 2021-08-10 at 10 42 47 AM" src="https://user-images.githubusercontent.com/34557516/128903005-990edb42-21b1-466a-b487-aa6521c5e5a7.png">

Manual testing steps:  
- Enter send flow on a custom network with native currency other than ETH
- Check that label in top left accurately describes the currency being sent.

Now:
<img width="1071" alt="Screen Shot 2021-08-10 at 12 05 15 PM" src="https://user-images.githubusercontent.com/34557516/128903229-c5744818-fc61-4703-82b6-5d7d1006cc41.png">
